### ISSUE TITLE
The Q button - Selection Window

### ISSUE THREAD

**Issue by AWxxx** [Unvanquished/Unvanquished#3150]
The Q button should not only open the selection window but also close it. The current setup disrupts the logic and slows down the game.

**Comment by DolceTriade** [comment]
agreed. I think there is a refactor of the menus. Can you please try the menus on the Bug Squash Server and see if they are better and report back? if so, then we can merge those.

**Comment by AWxxx** [comment]
There are two servers with that name. Q still only opens the window but doesn’t close it, on both of them. So its not solved.
